setup/raspberry-pi/ansible-playbook.yaml

```diff
@@ -156,10 +167,10 @@
       tags: jenkins
 
     - name: ccache | Install ccache
-      include: ../ansible-tasks/ccache.yaml version=3.3
+      include: ../ansible-tasks/ccache.yaml version=3.3.4
       tags: ccache
 
-    - include: ../ansible-tasks/git.yaml version=2.10.0
+    - include: ../ansible-tasks/git.yaml version=2.14.1
```

Comment:
Are we going to have to bump these manually going forwards (for things like https://github.com/nodejs/build/issues/831)?